web/pages/fast.svelte

```diff
@@ -8,15 +8,29 @@
     openedFastPost,
     fastMessages as t
   } from '@slowreader/core'
+  import { onMount } from 'svelte'
 
   import { getURL } from '../stores/router.js'
   import Button from '../ui/button.svelte'
   import Loader from '../ui/loader.svelte'
   import PostCard from '../ui/post-card.svelte'
   import TwoStepsPage from '../ui/two-steps-page.svelte'
+
+  let twoStepsPageInstance: TwoStepsPage
+
+  onMount(() => {
+    const unbindFast = fastPosts.subscribe(() => {
```

Comment:
You can make this code more compact by just returning `fastPosts.subscribe` result from callback.